pkgs/development/tools/hatch/default.nix

```diff
@@ -0,0 +1,69 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, python3
+, git
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "hatch";
+  version = "1.3.1";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
```

Comment:
Please use fetchPypi like https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/python-modules/hatchling/default.nix

Reply:
That doesn't matter. We often use fetchFromGitHub for various reasons.